mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -2828,6 +2828,14 @@ impl MmCoin for TendermintCoin {
         dex_fee_amount: DexFee,
         _stage: FeeApproxStage,
     ) -> TradePreimageResult<TradeFee> {
+        if dex_fee_amount.zero_fee() {
+            return Ok(TradeFee {
+                coin: self.ticker.clone(),
+                amount: MmNumber::default(),
+                paid_from_trading_vol: false,
+            });
+        }
```

Comment:
get_fee_to_send_taker_fee is legacy swap method
<img width="866" alt="image" src="https://github.com/user-attachments/assets/8120126e-a1e6-4c2e-bb0a-ef9846e22bb2" />